priv/repo/migrations/20241121160001_create_collection_items_id.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Lightning.Repo.Migrations.CreateCollectionItemsSerialId do
+  use Ecto.Migration
+
+  def change do
+    alter table(:collection_items) do
+      add :id, :bigint
+    end
+
+    create unique_index(:collection_items, [:collection_id, :id])
```

Comment:
Would we benefit from adding an order to this index? Since we're always sorted by id ascending?

Reply:
The idea here is to allow Postgres to go from one id to the next as a child node on the the BTREE once the pagination is always performed by collection.